### PR TITLE
Rubocop tidy

### PR DESCRIPTION
**Pull request by colinbruce**
Remove more of the rubocop:todos

### Number of offences ignored by ToDo
Start of session:  492
Lunchtime:          406
Hometime:          324
Saturday morning: 302
Reduced overlong lines from 145 to 5.

### CODE REVIEW

File: app/controllers/external_users/claims_controller.rb [hunk 1]
```diff
@@ -112,7 +113,8 @@ def unarchive
   def new
     load_offences_and_case_types
     build_nested_resources
-    track_visit({ url: 'external_user/%{type}/claim/new/page1', title: 'New %{type} claim page 1' }, claim_tracking_substitutions)
+    track_visit({ url: 'external_user/%{type}/claim/new/page1',
+                  title: 'New %{type} claim page 1' }, claim_tracking_substitutions)
```
Comment (jsugarman) [inline app/controllers/external_users/claims_controller.rb]: i'd put another linebreak before `claim_tracking_substitutions` for consistency with code below

File: app/interfaces/api/helpers/resource_helper.rb [hunk 2]
```diff
@@ -11,7 +11,8 @@ def claim_source
     end
 
     def claim_id
-      ::Claim::BaseClaim.active.find_by(uuid: params.claim_id).try(:id) || (raise ArgumentError, 'Claim cannot be blank')
+      err_msg = 'Claim cannot be blank'
+      ::Claim::BaseClaim.active.find_by(uuid: params.claim_id).try(:id) || (raise ArgumentError, err_msg)
```
Comment (jsugarman) [inline app/interfaces/api/helpers/resource_helper.rb]: alternative
```
claim = ::Claim::BaseClaim.active.find_by(uuid: params.claim_id)
raise ArgumentError,  'Claim cannot be blank' unless claim
claim&.id
```

Comment (jsugarman) [reply app/interfaces/api/helpers/resource_helper.rb]: feel free to ignore

File: app/interfaces/api/v1/claim_params_helper.rb [hunk 3]
```diff
@@ -6,7 +6,7 @@ module ClaimParamsHelper
     #
     params :common_params do
       optional :api_key, type: String, desc: 'REQUIRED: The API authentication key of the provider'
-      optional :creator_email, type: String, desc: 'REQUIRED: The ADP administrator account email address that uniquely identifies the creator of the claim.'
+      optional :creator_email, type: String, desc: I18n.t('api.v1.common_params.creator_email')
```
Comment (jsugarman) [inline app/interfaces/api/v1/claim_params_helper.rb:9]: This is the kind do of thing where we have a choice between doing it piecemeal, just to satisfy rubocop, or "all or nothing". Will leave it to your judgment

File: app/models/ability.rb [hunk 4]
```diff
@@ -1,6 +1,9 @@
 class Ability
   include CanCan::Ability
 
+  CLAIM_METHODS = %i[show show_message_controls messages edit update summary
```
Comment (jsugarman) [inline app/models/ability.rb]: Just a suggestion: `MANAGE_CLAIM_METHODS`

File: app/controllers/case_workers/admin/case_workers_controller.rb [hunk 5]
```diff
@@ -5,7 +5,8 @@ class CaseWorkers::Admin::CaseWorkersController < CaseWorkers::Admin::Applicatio
 
   def index
     @case_workers = CaseWorker.includes(:location).joins(:user)
-    @case_workers = @case_workers.where("lower(users.first_name || ' ' || users.last_name) ILIKE :term", term: "%#{params[:search]}%") if params[:search].present?
+    query = t('sql.query.name')
```
Comment (jsugarman) [inline app/controllers/case_workers/admin/case_workers_controller.rb]: Not sure about using translations for SQL. I'd be tempted to put it in a HEREDOC method in the controller or a helper if necessary. Or if it is a complex query then a `query object`

**Review by jsugarman — COMMENTED** [review]
Good stuff, a few minor suggestions and just one concern about putting SQL in translations. Does not seem the right place to put them. A HEREDOC method maybe??

File: app/interfaces/api/v1/claim_params_helper.rb (hunk 3 above)
Comment (colinbruce) [reply app/interfaces/api/v1/claim_params_helper.rb:9]: Piecemeal FTW! :) Also, added an issue to address this... at some point!

**Review by jsugarman — APPROVED** [review]
👍

**Pull request merged**